### PR TITLE
Merge for Issue #23

### PR DESCRIPTION
**Pull request by kryptodrex**

### CODE REVIEW

**Comment by kryptodrex** [comment]
Hi @evanradcliffe and @UjiMani, this is ready for review when you get a chance 😄. Brings entire Order management endpoints for saving orders to DB and getting them out, and also updating travelers on an order.

File: trainsim-client/src/components/GoogleLoginField.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default class GoogleLoginField extends React.Component<{}, LoginState> {
     loginSuccess(response: any) {
         const email = response.profileObj.email;
 
-        fetch(`/api/user?email=${email}`)
+        fetch(`/api/users?email=${email}`, {method: "POST"})
```
Comment (evanradcliffe) [inline trainsim-client/src/components/GoogleLoginField.tsx:26]: why POST here? consolidating javalin route?

Comment (kryptodrex) [reply trainsim-client/src/components/GoogleLoginField.tsx:26]: Yeah, it is a POST method to create and/or get the user data. The old method was just a GET, which didn't make much sense if you were creating a user

Comment (evanradcliffe) [reply trainsim-client/src/components/GoogleLoginField.tsx:26]: oh yea, I read this as getting user info not creating 🤦

**Review by evanradcliffe — APPROVED** [review]

**Pull request merged**